store/src/sqlite/model.rs

```diff
@@ -709,8 +715,8 @@ where
             .await?;
         Ok(row
             .into_iter()
-            .map(|row| EventId::from(row.get::<Vec<u8>, _>(0)))
-            .collect())
+            .map(|row| EventId::try_from(row.get::<Vec<u8>, _>(0)))
```

Comment:
Would `.map(|row| EventId::try_from(row.get::<Vec<u8>, _>(0))?)` prevent us from needing to change the next line?

Reply:
No, for the same reason as above, the map closure has to return a Result<_> in either case and we have to propogate that Result to the outer function.